apps/coscrad-frontend/src/components/footer/footer.spec.tsx

```diff
@@ -40,3 +40,48 @@ describe('Footer', () => {
         expect(screenRes).toBeTruthy();
     });
 });
+
+describe('when an email is provided', () => {
+    const dummyConfigurableContent = getDummyConfigurableContent();
+
+    it('should render the email icon and address', () => {
+        const email = 'johndoe@johndoe.com';
+        const myConfigurableContent = { ...dummyConfigurableContent, email };
+
+        renderWithProviders(
+            <MemoryRouter>
+                <Footer />
+            </MemoryRouter>,
+            {
+                contentConfig: myConfigurableContent,
+            }
+        );
+
+        const searchPattern = new RegExp(email);
+        const screenRes = screen.getByText(searchPattern);
+
+        expect(screenRes).toBeTruthy();
+    });
+});
+
+describe('when an email is not provided', () => {
+    const dummyConfigurableContent = getDummyConfigurableContent();
+
+    it('does not render email icon when email is empty', () => {
+        const email = '';
+        const myConfigurableContent = { ...dummyConfigurableContent, email };
+
+        renderWithProviders(
+            <MemoryRouter>
+                <Footer />
+            </MemoryRouter>,
+            {
+                contentConfig: myConfigurableContent,
+            }
+        );
+
+        const screenRes = screen.queryByTestId('email-icon');
```

Comment:
Don't remove the check for the actual email address. You can do multiple assertions in one test case since they are so closely related.